src/emulator/pubsubEmulator.ts

```diff
@@ -25,7 +25,7 @@ interface Trigger {
 }
 
 export class PubsubEmulator implements EmulatorInstance {
-  pubsub: PubSub;
+  private _pubsub: PubSub | undefined;
```

Comment:
it's redundant to prefix private fields with `_`. Please omit.

Reply:
The getter is also called `pubsub` and we'd have a naming conflict if `_` were to be removed.